package/yast2-migration.changes

```diff
@@ -7,11 +7,17 @@ Fri Oct  7 10:30:08 UTC 2016 - igonzalezsosa@suse.com
 - 3.1.0.13
 
 -------------------------------------------------------------------
+Thu Nov 26 13:17:47 UTC 2015 - stephan.barth@suse.com
+
+- Added Supplements for yast2-registration to ensure automatic
+  installation
+- 3.1.0.12
+
+-------------------------------------------------------------------
 Thu Oct  8 13:26:01 UTC 2015 - lslezak@suse.cz
 
 - fixed crash in the migration proposal (do not use "return"
   for returning the client result value) (bsc#949484)
-- 3.1.0.12
```

Comment:
I'd rather keep the original changes to avoid problems in IBS (changing the old items), see https://build.suse.de/package/view_file/SUSE:SLE-12:Update/yast2-migration/yast2-migration.changes?expand=1